robot_skills/src/robot_skills/arms.py

```diff
@@ -4,6 +4,7 @@
 import rospy
 import std_msgs.msg
 import PyKDL as kdl
+import smach
```

Comment:
It still feels weird to have `smach` in `robot_skills`.

If it stays here, smach should be added as dependency in `package.xml` and `CMakeLists.txt`

Reply:
The easy way around this would be change the position of the collect_arm_requirements def. @PetervDooren also had a remark about this so I will see about moving it somewhere else.